back/engines/commercial/insights/app/controllers/insights/web_api/v1/stats_inputs_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Insights
+  module WebApi::V1
+    class StatsInputsController < ::ApplicationController
+      def inputs_count
+        @result = Insights::InputsFinder.new(view, counts_params).execute
+
+        render json: { count: @result.count }
```

Comment:
(Tiny detail: don't think an instance variable is needed here.)

```suggestion
        inputs = Insights::InputsFinder.new(view, counts_params).execute

        render json: { count: inputs.count }
```